lib/util.js

```diff
@@ -118,7 +114,7 @@ function wrapRouteHandlers(route, app) {
         path = app.metrics.normalizeName(path || 'root');
         routerLayer.route.stack.forEach((layer) => {
             const origHandler = layer.handle;
-            layer.handle = function(req, res, next) {
+            layer.handle = function (req, res, next) {
```

Comment:
Nit: use `() => {}` notation for consistency.

Reply:
Done